File: scripts/config_loader.py
import yaml
from pathlib import Path

def load_config(path="config/config.yml"):
    with open(path, "r") as f:
        return yaml.safe_load(f)

def load_species_params(path="config/species_params.yml"):
    with open(path, "r") as f:
        return yaml.safe_load(f)

File: scripts/config_loader.R
library(yaml)

load_global_config <- function() {
  yaml::read_yaml("config/config.yml")
}

load_stats_params <- function() {
  yaml::read_yaml("config/stats_params.yml")
}

load_species_params <- function() {
  yaml::read_yaml("config/species_params.yml")
}
